modules/options/src/main/scala/scala/build/actionable/ActionablePreprocessor.scala

```diff
@@ -0,0 +1,21 @@
+package scala.build.actionable
+
+import scala.build.Ops._
+import scala.build.errors.{BuildException, CompositeBuildException}
+import scala.build.options.BuildOptions
+
+case object ActionablePreprocessor {
+  val actionableHandlers = Seq(
```

Comment:
This should be typed as `Seq[ActionableHandler]`, so that the details of `ActionableDependencyHandler` don't leak here:
```suggestion
  val actionableHandlers = Seq[ActionableHandler](
```
(this should work fine with the type bound suggestion above)